From 9a2fa4a719df870296f8559bbf775696b49847c5 Mon Sep 17 00:00:00 2001
From: Don Penney <don.penney@windriver.com>
Date: Fri, 7 Feb 2020 14:09:28 -0500
Subject: [PATCH] Add support for https and IPv6 to anaconda-preexec

Include the source patch that adds support for https and IPv6.

Signed-off-by: Don Penney <don.penney@windriver.com>
---
 SPECS/anaconda.spec | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/SPECS/anaconda.spec b/SPECS/anaconda.spec
index c482f38..8fea2fa 100644
--- a/SPECS/anaconda.spec
+++ b/SPECS/anaconda.spec
@@ -16,10 +16,12 @@ Patch100: centos-l10n.patch
 Patch101: 0001-branding-syspurpose-doesn-t-make-sense-for-CentOS.patch
 Patch102: 0002-add-centos-background-color.patch
 
+# StarlingX
 Patch10001: 0001-TIS-Progress-and-error-handling.patch
 Patch10002: 0002-revert-7.4-grub2-efi-handling.patch
 Patch10003: 0003-Set-default-hostname-to-localhost.patch
 Patch10004: 0004-Cache-server-ip-in-etc-hosts.patch
+Patch10005: 0005-Add-support-for-IPv6-and-https-to-anaconda-preexec.patch
 
 # To generate Source0 do:
 # git clone https://github.com/rhinstaller/anaconda
-- 
2.7.4

